server/src/main/java/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -2139,14 +2139,14 @@ public boolean startRemoteAccessVpn(final Network network, final RemoteAccessVpn
             }
 
             Answer answer = cmds.getAnswer("users");
-            if (!answer.getResult()) {
+            if (answer != null && !answer.getResult()) {
```

Comment:
You can use `org.apache.commons.lang3.BooleanUtils.toBooleanObject(String, String, String, String)` here